sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/GlobalThroughputControlConfig.java

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
+import com.azure.cosmos.util.Beta;
+
+import java.time.Duration;
+
+/**
+ * This configuration is used for throughput global control mode.
+ * It contains configuration about the extra container which will track all the clients throughput usage for a certain control group.
+ */
+@Beta(value = Beta.SinceVersion.V4_13_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+public class GlobalThroughputControlConfig {
+    private final CosmosAsyncContainer controlContainer;
+    private final Duration controlItemRenewInterval;
+    private final Duration controlItemExpireInterval;
+
+
```

Comment:
nit: you have 2 empty lines, change to 1 empty line plz

Reply:
removed